frontend/src/scenes/instance/AsyncMigrations/AsyncMigrations.tsx

```diff
@@ -61,158 +66,185 @@ export function AsyncMigrations(): JSX.Element {
+                                        status="stealth"
+                                        onClick={() => forceStopMigrationWithoutRollback(asyncMigration)}
+                                        fullWidth
+                                    >
+                                        Stop
+                                    </LemonButton>
+                                </>
+                            }
+                        />
+                    ) : status === AsyncMigrationStatus.CompletedSuccessfully ? (
+                        <></>
+                    ) : status === AsyncMigrationStatus.Errored ? (
+                        <More
+                            overlay={
+                                <>
+                                    <LemonButton
+                                        status="stealth"
+                                        onClick={() => resumeMigration(asyncMigration)}
+                                        fullWidth
+                                    >
+                                        Resume
+                                    </LemonButton>
+                                    <LemonButton
+                                        status="stealth"
+                                        onClick={() => rollbackMigration(asyncMigration)}
+                                        fullWidth
+                                    >
+                                        Rollback
+                                    </LemonButton>
+                                </>
+                            }
+                        />
+                    ) : status === AsyncMigrationStatus.RolledBack ? (
+                        <Tooltip title="Restart">
+                            <LemonButton
+                                status="stealth"
+                                icon={<IconReplay />}
+                                onClick={() => triggerMigration(asyncMigration)}
+                                fullWidth
                             />
-                        ) : status === AsyncMigrationStatus.RolledBack ? (
-                            <Tooltip title="Restart">
-                                <LemonButton
-                                    status="stealth"
-                                    icon={<IconReplay />}
-                                    onClick={() => triggerMigration(asyncMigration)}
-                                    fullWidth
-                                />
-                            </Tooltip>
-                        ) : status === AsyncMigrationStatus.Starting ? (
-                            <Spinner size="sm" />
-                        ) : null}
-                    </div>
-                )
-            },
+                        </Tooltip>
+                    ) : null}
+                </div>
+            )
+        },
+    }
+
+    const minVersionColumn: AsyncMigrationColumnType = {
+        title: 'Min PostHog Version',
```

Comment:
nit: In user-facing things like these, let's maybe either use full names or explicitly mark the abbreviation. Also other columns are using sentence case 

```suggestion
        title: 'Min. PostHog version',
```